server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -1512,6 +1520,7 @@ private boolean alterTimeSeries(AlterTimeSeriesPlan alterTimeSeriesPlan)
 
   public boolean setStorageGroup(SetStorageGroupPlan setStorageGroupPlan)
       throws QueryProcessException {
+    AUDIT_LOGGER.info("set storage group to {}", setStorageGroupPlan.getPaths());
     PartialPath path = setStorageGroupPlan.getPath();
```

Comment:
getPath or getPaths?

Reply:
The return value is same, but getPath will be better. I'll change it